src/hooks/connectors/useHasConnectors.ts

```diff
@@ -0,0 +1,34 @@
+import { useMemo } from 'react';
+import { TABLES } from 'services/supabase';
+import { connectorHasRequiredColumns } from 'utils/connector-utils';
+import { useQuery, useSelect } from '../supabase-swr';
+import { ConnectorsExist, CONNECTORS_EXIST_QUERY } from './shared';
+
+// TODO (connectors store) - this is temporary
+// We used to check if connectors exist with a query that returned a bunch
+//      of data and sorting. However, we then go and fetch that again a second
+//      later. So this query is super small to reduce the amount of data and just
+//      making sure there are connectors.
+// We should just make a store that fetches all the needed connectors once
+//      then always pull from that. The store could also cache the schemas, etc.
+//      when the user selects a connector.
+function useHasConnectors(protocol: string | null) {
```

Comment:
`useConnectorsExist` and `requiredConnectorColumnsExist` read more easily IMO.

Reply:
I love `requiredConnectorColumnsExist`! I could not think of how to state this before and keep the work `Connector` in there haha... I'll update.